### PR TITLE
fix: FastAPI is now fully functional, unless noted otherwise

### PR DESCRIPTION
**Pull request by jmargaret2**
### Summary
_Provide an overview..._

### Details
As proven in the testing prior to our demo, FastAPI fully works now

### Checks
- [ x] Tested changes
- [x ] Attached Logs

### Team to Review
sylveon

### Reference to the issue
bug fixes

### CODE REVIEW

**Comment by KevZhe** [comment]
we made it to the promised land

**Review by SushieeK — APPROVED** [review]
Yaay! Great Work. LGTM.

**Review by KevZhe — APPROVED** [review]
great work

File: src/gradescopeapi/classes/account.py [hunk 1]
```diff
@@ -121,6 +126,7 @@ def get_assignments(self, course_id: str) -> List[Assignment]:
             "You must be logged in to access this page.": if no user is logged in
         """
         course_endpoint = f"https://www.gradescope.com/courses/{course_id}"
+        print(course_endpoint)
```
Comment (KevZhe) [inline src/gradescopeapi/classes/account.py:129]: debug statement still here

File: src/gradescopeapi/classes/account.py [hunk 2]
```diff
@@ -107,6 +111,7 @@ def get_course_users(self, course_id: str) -> List[str]:
 
             return users
         except Exception as e:
+            print(e)
```
Comment (KevZhe) [inline src/gradescopeapi/classes/account.py:114]: and here

**Comment by jmargaret2** [comment]
> we made it to the promised land

https://media.giphy.com/media/blSTtZehjAZ8I/giphy.gif?cid=790b76115o1q9lbvhh1yyq126yd6n0nrkwpuji3qpnqph8xj&ep=v1_gifs_search&rid=giphy.gif&ct=g

File: src/gradescopeapi/classes/account.py (hunk 1 above)
Comment (jmargaret2) [reply src/gradescopeapi/classes/account.py:129]: fixed

File: src/gradescopeapi/classes/account.py (hunk 2 above)
Comment (jmargaret2) [reply src/gradescopeapi/classes/account.py:114]: fiex

**Pull request merged**